static/specification.html

```diff
@@ -138,605 +336,555 @@ <h2>
+        </p>
+        <p>
+          The appropriate times to <span data-x=
+          "fetch and process the linked resource">fetch and process</span> this
+          type of link is:
+        </p>
+        <ul>
+          <li>
+            <p>
+              When the <span>external resource link</span> is created on a
+              <code>link</code> element that is already <span>browsing-context
+              connected</span>.
+            </p>
+          </li><!-- e.g. rel="" changed, href="" set... -->
+          <li>
+            <p>
+              When the <span>external resource link</span>'s <code>link</code>
+              element <span>becomes browsing-context connected</span>.
+            </p>
+          </li>
+          <li>
+            <p>
+              When the <code data-x="attr-link-href">href</code> attribute of
+              the <code>link</code> element of an <span>external resource
+              link</span> that is already <span>browsing-context
+              connected</span> is changed.
+            </p>
+          </li>
+          <li>
+            <p>
+              When the <code data-x="attr-link-disabled">disabled</code>
+              attribute of the <code>link</code> element of an <span>external
+              resource link</span> that is already <span>browsing-context
+              connected</span> is set, changed, or removed.
+            </p>
+          </li>
+          <li>
+            <p>
+              When the <code data-x="attr-link-crossorigin">crossorigin</code>
+              attribute of the <code>link</code> element of an <span data-x=
+              "external resource link">external resource link</span> that is
+              already <span>browsing-context connected</span> is set, changed,
+              or removed.
+            </p>
+          </li>
+          <li>
+            <p>
+              When the <code data-x="attr-link-type">type</code> attribute of
+              the <code>link</code> element of an <span>external resource
+              link</span> that is already <span>browsing-context
+              connected</span> is set or changed to a value that does not or no
+              longer matches the <span data-x="Content-Type">Content-Type
+              metadata</span> of the previous obtained external resource, if
+              any.
+            </p>
+          </li>
+          <li>
+            <p>
+              When the <code data-x="attr-link-type">type</code> attribute of
+              the <code>link</code> element of an <span>external resource
+              link</span> that is already <span>browsing-context
+              connected</span>, but was previously not obtained due to the
+              <code data-x="attr-link-type">type</code> attribute specifying an
+              unsupported type, is set, removed, or changed.
+            </p>
+          </li>
+        </ul>
+        <p>
+          A user agent MUST NOT <span>delay the load event</span> for this link
```

Comment:
It shouldn't be easy for a website to figure out a user is able to do web monetization, but has chosen not to do so. We think the event shouldn't fire when monetization is disabled on a page. And that's because the user agent shouldn't process the payment pointer in the first place when monetization is disabled on a page.

Reply:
> It shouldn't be easy for a website to figure out a user is able to do web monetization, but has chosen not to do so.

User comes across site with automatic "zero friction" WM enabled 
User Agent sends packets of money
User decides they don't want to support the site and disables automatic WM for the site
Site sees packets on initial page loads session (cookies/whatever ) 
Site doesn't see packets on subsequent page loads  

It's not going to be that hard 

Given the tools (we've never offered them previously) there could be a subset of WM users who disable automatic WM globally and only tip. 

Are we saying when automatic payment is disabled, the load event doesn't fire?

Reply:
I suggest we leave this to UA vendors to figure out based on their own privacy policies and implementation experience. They may provide us with extra detail we need to add to the spec.